source/UberCore/Authentication/OAuthEndpoint.swift

```diff
@@ -59,13 +60,23 @@ public enum OAuth: APIEndpoint {
             var components = URLComponents()
             components.queryItems = query
             return components.query?.data(using: String.Encoding.utf8)
+        case .par(let clientID, let loginHint, let responseType):
+            let loginHintString = (try? JSONSerialization.data(withJSONObject: loginHint))?.base64EncodedString() ?? ""
+            let query = queryBuilder(
+                ("client_id", clientID),
+                ("response_type", responseType.rawValue),
+                ("login_hint", loginHintString)
+            )
+            var components = URLComponents()
+            components.queryItems = query
+            return components.query?.data(using: String.Encoding.utf8)
         default:
             return nil
         }
     }
 
     static var regionHost: String {
-        return "https://login.uber.com"
+        return "https://auth.uber.com"
```

Comment:
This changes the host for all the requests I assume. Is this safe?

Reply:
This should be ok from our testing. But I will confirm with backend.

Reply:
Verified that this host is good for all endpoints in OAuth